data/transactions/logic/eval.go

```diff
@@ -682,6 +682,22 @@ func opPlus(cx *evalContext) {
 	cx.stack = cx.stack[:last]
 }
 
+func opPluswImpl(x, y uint64) (carry uint64, sum uint64) {
```

Comment:
This is only used in one place, why not just be inline?

Reply:
I think the idea is to have `TestPluswImpl` but the test is not there.

Reply:
Ah, yeah, oops--forgot to add that test. Will do that.